src/Installers/Windows/Wix.targets

```diff
@@ -64,18 +64,12 @@
 
   <Target Name="CopyToArtifactsDirectory"
           Condition=" '$(IsShipping)' == 'true' AND '$(SkipCopyToArtifactsDirectory)' != 'true' "
-          AfterTargets="Build" BeforeTargets="GenerateChecksums">
```

Comment:
Why is this changed?

Reply:
This is back to the way the file was before my first checksum change, except with 'AfterTargets=Build' changed to 'BeforeTargets=Build'. Doug suggested the change since `Build` is a no-op target that runs at the end of the project build, and it's more deterministic to hook into it w/ a `Before` than an `After`

Reply:
Ah I missed that comment.